test/integration/test_utils.js

```diff
@@ -137,14 +137,20 @@ const mockClipboard = async pages => {
 };
 exports.mockClipboard = mockClipboard;
 
-const getSerialized = page =>
-  page.evaluate(() => {
+const getSerialized = async (page, filter = undefined) => {
```

Comment:
Could we perhaps just use regular function notation here, and below, instead?
```suggestion
async function getSerialized(page, filter = undefined) {
```